include/nlohmann/detail/meta/can_call_std.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <nlohmann/detail/meta/detected.hpp>
+
+// source: https://stackoverflow.com/a/26745591
+
+#define CAN_CALL_STD_FUNC_IMPL(std_name)                                          \
```

Comment:
This was replaced with `NLOHMANN_CAN_CALL_STD_FUNC_IMPL` right?

Reply:
Should be good